### ISSUE TITLE
Support for flags based on timezone codes

### ISSUE THREAD

**Issue by kry0sc0pic** [moazelsawaf/dash_flags#2]
Would it possible to add flags using the timezone codes from the [IANA Timezone Database](https://www.iana.org/time-zones)?

for example:
```dart
TimezoneFlag(
   timezone: 'America/Los_Angeles',
   height: 50,
)
```

thank you.

**Comment by moazelsawaf** [comment]
Nice idea 💙

I have studied and implemented it in `v0.1.0` and it is live on [pub.dev](https://pub.dev/packages/dash_flags) now!

Kindly checkout the docs 📃

Thank you so much for suggesting this idea 🙏🏻

**Comment by kry0sc0pic** [comment]
Thank you ❤️
